src/util/string_utils.cpp

```diff
@@ -156,13 +156,16 @@ std::string escape_non_alnum(const std::string &to_escape)
   std::ostringstream escaped;
   for(auto &ch : to_escape)
   {
+    const auto uch = static_cast<unsigned char>(ch);
```

Comment:
Actually had to look this up, the standard actually guarantees this is well defined and behaves like a sane person would expect! A surprise to be sure, but a welcome one.